pages/doc/logging_proxy_configurations.md

```diff
@@ -109,6 +109,15 @@ We've added the following configuration properties for logs to the already exist
 <br/>Example: 2000</td>
 </tr>
 <tr>
+<a name="pushFlushMaxLogs"></a>
+<td>pushFlushMaxLogs</td>
+<td markdown="span"> The maximum size of the log that is sent to the server. If the log exceeds the value mentioned here, the log is not sent to the server.
+<br/> Default: 4 MB
```

Comment:
@shavidissa This value should be in the bytes, I think it is the maximum size of a log payload in a single flush in bytes.